src/content/tools/pub/publishing.md

```diff
@@ -37,16 +37,15 @@ upgrade yet.
 For already published packages that are no longer relevant or being maintained,
 you can [mark them as discontinued](#discontinue).
 
-## Preparing to publish
+## Prepare to publish
 
-When publishing a package, it's important to follow the [pubspec
-format][pubspec] and
+When publishing a package, follow the [pubspec format][pubspec] and
 [package layout conventions][].
-Some of these are required in order for others to be able to use your package.
-Others are suggestions to help make it easier for users to understand and work
-with your package. In both cases, pub tries to help you by pointing out what
-changes will help make your package play nicer with the Dart ecosystem. There
-are a few additional requirements for uploading a package:
+Some of these are required for others to use your package.
+Others are suggestions to simplify concepts other users to understand and work
```

Comment:
"simplify concepts other users to understand" sounds strange to me. Is this right?